op-program/Dockerfile.repro

```diff
@@ -41,6 +41,9 @@ RUN /app/cannon/bin/cannon load-elf --path /app/op-program/bin/op-program-client
 RUN /app/cannon/bin/cannon run --proof-at '=0' --stop-at '=1' --input /app/op-program/bin/prestate.json --meta "" --proof-fmt '/app/op-program/bin/%d.json' --output ""
 RUN mv /app/op-program/bin/0.json /app/op-program/bin/prestate-proof.json
 
+RUN /app/cannon/bin/cannon run --proof-at '=0' --stop-at '=1' --input /app/op-program/bin/prestate-mt.bin.gz --meta "" --proof-fmt '/app/op-program/bin/%d-mt.json' --output ""
```

Comment:
There's [a command above](https://github.com/ethereum-optimism/optimism/blob/28431b524635550a0193e96264b80f102f4d3852/op-program/Dockerfile.repro#L38) to generate the single-threaded prestate, do we need a similar command to generate `prestate-mt.bin.gz`?

Reply:
Yes we do. Good catch. 
It appeared to work on my machine because I already had a MT proof cached.